docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[searchable-snapshots]]
+== {search-snaps-cap}
+
+beta::[]
+
+{search-snaps-cap} let you reduce your operating costs by using
+<<snapshot-restore, snapshots>> for resiliency rather than maintaining
+<<scalability,replica shards>> within a cluster. When you mount an index from a
+snapshot as a {search-snap}, {es} copies the index shards to local storage
+within the cluster. This ensures that search performance is comparable to
+searching any other index, and minimizes the need to access the snapshot
+repository. Should a node fail, shards of a {search-snap} are automatically
+recovered from the snapshot repository.
+
+This can result in significant cost savings. With {search-snaps}, you may be
+able to halve your cluster size without increasing the risk of data loss or
+reducing the amount of data you can search. Because {search-snaps} rely on the
+same snapshot mechanism you use for backups, they have a minimal impact on your
+snapshot repository storage costs.
+
+[discrete]
+[[using-searchable-snapshots]]
+=== Using {search-snaps}
+
+Searching a {search-snap} is the same as searching any other index. Search
```

Comment:
Should this be "Searching a searchable snapshot index" to be consistent with glossary. I think it reads better too.

Same comment goes for a number of the "{search-snap}" mentions throughout.

Reply:
Good point, I've added a few "index" or "shard" nouns throughout in 990707b6758d4c82243aec92cb582e34daa828ca.